contrib/testgen/gen_key_io_test_vectors.py

```diff
@@ -4,14 +4,9 @@
 # file COPYING or http://www.opensource.org/licenses/mit-license.php.
 '''
 Generate valid and invalid base58 address and private key test vectors.
-
-Usage:
-    PYTHONPATH=../../test/functional/test_framework ./gen_key_io_test_vectors.py valid 50 > ../../src/test/data/key_io_valid.json
-    PYTHONPATH=../../test/functional/test_framework ./gen_key_io_test_vectors.py invalid 50 > ../../src/test/data/key_io_invalid.json
 '''
 # 2012 Wladimir J. van der Laan
```

Comment:
@laanwj This says your name, which makes you qualified to review this? :grimacing:

Reply:
I definitely don't remember writing this :sweat:

Reply:
Your name is removed in current master, but you are still welcome to review